scss/sections/_footer.scss

```diff
@@ -1,303 +1,59 @@
-	border-radius: 6px;
-}
-
-.footer-form-widget form .input-group input[type="email"] {
-	width: 100%;
-	border: none;
-	margin: 0px;
-	font-size: 1.25rem;
-	line-height: 30px;
-	padding: 14px 16px;
-	border-radius: 6px;
-	color: #7b7b7b;
-	font-weight: 400;
-}
-
-.footer-form-widget form .input-group button.newsletter__submit {
-	width: 130px;
-	font-size: 1.25rem;
-	font-weight: 700;
-	color: #fff;
-	position: absolute;
-	top: 0px;
-	right: 0px;
-	background: #f4436c;
-	padding: 14px 10px;
-	line-height: 30px;
-	border: none;
-	border-radius: 6px;
-}
-
-form.footer-search-form {
-	width: 100%;
-	display: inline-block;
-	position: relative;
-	margin-top: 5px;
-}
-
-form.footer-search-form input[type='text'],
-form.footer-search-form input[type='search'] {
-	width: 100%;
-	height: 40px;
-	border-radius: 6px;
-	padding: 0 42px 0 16px;
-	border: none;
-	color: #adadad;
-	font-size: 1rem;
-	line-height: normal;
-	font-style: italic;
-}
-
-form.footer-search-form input[type='search'] {
-	line-height: 20px;
-}
-
-form.footer-search-form button {
-	padding: 0;
-	border: none;
-	margin: 0;
-	line-height: 0;
-	position: absolute;
-	top: 0px;
-	right: 0;
-	background: none;
-	height: 40px;
-	width: 40px;
-}
-
-.footer_bottom {
-	padding: 10px 0;
-	border-top: 1px solid #fff;
-}
-
-.footer_bottom p {
-	font-size: 1rem;
-	color: #fff;
-}
-
-@include media-breakpoint-down(lg) {
-	.footer-widget ul.list-unstyled {
-		margin-bottom: 1rem;
-	}
-
-	.footer-widget ul.list-unstyled li a {
-		font-size: 0.875rem;
-		line-height: 1.25rem;
-	}
-
-	.footer-widget .footer-widget-text {
-		padding: 4px 0 10px 0;
-	}
-
-	.footer-widget .footer-widget-text p {
-		font-size: 0.875rem;
-		line-height: 1.25rem;
-	}
-
-	.footer-widget .footer-widget-small-text {
-		padding: 5px 0 15px 0;
-	}
-
-	.footer-widget .footer-widget-small-text p {
-		font-size: 0.875rem;
-		line-height: 1.25rem;
-	}
-
-	.footer-form-widget {
-		margin-bottom: 10px;
-	}
-
-	.footer-form-widget form {
-		width: 100%;
-		display: inline-block;
-	}
-
-	.footer-form-widget form .input-group {
-		padding-right: 130px;
-		margin: 0px;
-		background: #fff;
-		border-radius: 6px;
-	}
-
-	.footer-form-widget form .input-group input[type="email"] {
-		font-size: 1rem;
-	}
-
-	.footer-form-widget form .input-group button.newsletter__submit {
-		width: 130px;
-		font-size: 1rem;
-	}
-
-	form.footer-search-form button {
-		top: 0px;
-	}
-}
-
-@include media-breakpoint-down(sm) {
-	.footer-widget ul.list-unstyled li {
-		margin-bottom: 4px;
-	}
-
-}
-
-.footer-container .navbar-nav .nav-link {
-	color: #fff;
-}
-
-.footer-container .navbar-nav .nav-item {
-	padding: 0px;
-}
-
-.footer-container h3 {
-	font-weight: 700;
-	font-size: 2rem;
-}
-
-footer p {
-	line-height: 1.3;
-}
-
-.footer_bottom p svg {
-	width: 18px;
-	margin-left: 10px;
-	margin-top: -7px;
-}
+  background-color: $secondary;
+  color: $white;
+
+  a {
+    color: $white;
+    font-size: 1rem;
+  }
+
+  .sni {
+    font-size: 1rem;
```

Comment:
unused too, please keep consistency always. even if need custom font-size we should always evaluate if it's true or just a design error.